G/Gnuplot/build_tarballs.jl

```diff
@@ -43,14 +45,30 @@ esac
 
 make -C src -j${nproc}
 make -C src install
+
+if [[ "${target}" == *-mingw* ]]; then
+    suffix='.exe'
```

Comment:
exeext does this already: https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables